lib/core/cardano-wallet-core.cabal

```diff
@@ -104,6 +104,7 @@ test-suite unit
     , aeson
     , aeson-qq
     , async
+    , bifunctors
```

Comment:
use `base` bifunctors

Reply:
I tried to remove it, but this package contains `Data.Bifunctor.TH.deriveBifunctor`, which I don't think `base` provides.